src/domain/space/space/space.service.authorization.ts

```
@@ -100,69 +102,73 @@ export class SpaceAuthorizationService {
+          // Pick up the parent authorization
+          const parentAuthorization =
+            this.getParentAuthorizationPolicyOrFail(space);
+          space.authorization =
+            this.authorizationPolicyService.inheritParentAuthorization(
+              space.authorization,
+              parentAuthorization
+            );
+          space.authorization.anonymousReadAccess = true;
+        }
+        // For subspace, the parent space admins credentials should be allowed to delete
+        const parentSpaceCommunity = space.parentSpace?.community;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)